components/tikv_util/src/lib.rs

```diff
@@ -50,6 +50,7 @@ pub mod logger;
 pub mod lru;
 pub mod metrics;
 pub mod mpsc;
+pub mod read_pool;
```

Comment:
Is it still suitable to be called "read_pool"?

Reply:
Yes, I will give another PR to rename it to `thread_pool` and remove the origin `threadpool.rs`.